llvm/test/tools/UpdateTestChecks/update_test_checks/Inputs/filter-all-after.ll

```diff
@@ -0,0 +1,17 @@
+; RUN: opt < %s -S | FileCheck %s
+
+define i32 @func({i32, i32} %x, i32 %y, i1 %cond) {
```

Comment:
Could you add a test with another function, to check that the filter applies per-function only?